### PR TITLE
SD-3745 Import "Stats HTTP API" section

### PR DESCRIPTION
**Pull request by seewerah**
### What Changed

Added a new "Stats HTTP API" section to "Momentum 4.x" with three subsections:

- Summary
- Mail Queue
- Queues Sizes

### How To Test or Verify

Just build and run it. The new section is seen by reaching the `/momentum/4/http-api-stats` endpoint.

#### All PRs Checklist

- [x] Give your pull request a meaningful name.
- [x] Use lowercase filenames.
- [x] Apply at least one team label according to which team is the content expert (ie. `team-FE` or `team-SAZ`)
- [x] Pull request approval from the FE team or content experts (see label applied above) that isn't the content creator.

#### Content Changes Checklist

- [x] Check that your article looks correct in the preview here or in a [Netlify deploy preview](https://app.netlify.com/sites/support-docs/deploys).
- [x] Check the links in your article.
- [x] Check the images in your article (if there are any)
- [x] Check to make sure you are using markdown appropriately as outlined in `examples/article.md` in the root of the project directory and on the momentum doc's [preface article](https://support.sparkpost.com/momentum/4/4-preface)
- [ ] Check to make sure the [Copy and Tone Guidelines are followed](https://docs.google.com/document/d/1dej9J7N9M8lcbJXnT9kxyNB_EFBPHIBLZBbHaxRWqhQ/edit).

### CODE REVIEW

File: content/momentum/4/console-commands/4-mailq.md [hunk 1]
```diff
@@ -28,4 +28,8 @@ default/#mmove                                   gmail.com       0       0
 
 The `Group/Binding` column shows the group and the binding that a domain belongs to. The `default/#move` entry under the `Group/Binding` column is the cluster message movement binding in the `default` binding. This is a virtual binding for moving messages between nodes. `AQ` shows the items in the active queue and `DQ` shows the items in the delayed queue.
 
-Use this command with the `--domain` *`domain_name`* argument to show the statistics for a specific domain. Any number of domains may be specified. The `--binding` option is used in the same way. Limit the output by using the --limit *`number`* option. The default value for `--limit` is `20`.
\ No newline at end of file
+Use this command with the `--domain` *`domain_name`* argument to show the statistics for a specific domain. Any number of domains may be specified. The `--binding` option is used in the same way. Limit the output by using the --limit *`number`* option. The default value for `--limit` is `20`.
+
+## Attention
+
+This command is unsafe to be executed in production and should be avoided. It is recommended to use the `mailq` HTTP endpoint instead.
```
Comment (cjarnold) [inline content/momentum/4/console-commands/4-mailq.md:35]: The command is okay to run occasionally.  So I think we should reword it to be less scary.  Something like:

On a busy system, if this command is run frequently it may result in a Momentum watchdog restart.  If mailq data is needed multiple times per hour then it is recommended to use the `mailq` HTTP endpoint instead.

Also, can we link to the new mailq HTTP doc?

File: content/momentum/4/console-commands/summary.md [hunk 2]
```diff
@@ -207,4 +207,8 @@ Use the `with-mobile` option to display the status of SMPP and MM7 message queue
 <a name="idp9306560"></a> 
 ## See Also
 
-[binding summary](/momentum/4/console-commands/binding-summary), [summary reset](/momentum/4/console-commands/summary-reset)
\ No newline at end of file
+[binding summary](/momentum/4/console-commands/binding-summary), [summary reset](/momentum/4/console-commands/summary-reset)
+
+## Attention
+
+This command is unsafe to be executed in production and should be avoided. It is recommended to use the `summary` HTTP endpoint instead.
```
Comment (cjarnold) [inline content/momentum/4/console-commands/summary.md:214]: Recommend same rewording as mentioned above for mailq.

File: content/momentum/4/http-api-stats/queues-mailq.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+## Synopsis
+
+`GET /stats/queues/mailq`
+
+`GET /stats/queues/mailq?domain=some.domain`
+
+`GET /stats/queues/mailq?binding=ip_1.2.3.4`
+
+`GET /stats/queues/mailq?ip=1.2.3.4`
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/queues-mailq.md:20]: We should remove this example of `ip` query parameter.  on prem would always be referencing "binding" instead of ip.

File: content/momentum/4/http-api-stats/queues-mailq.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+      "transfails": 0,
+      "deliveries": 0
+    }
+  ]
+}
+```
+
+The `group` and `binding` fields shows the group and the binding that a domain belongs to. `aq` shows the items in the active queue and `aq` shows the items in the delayed queue. The number of receptions, permanent failures, transient failures, and deliveries are shown in the `receptions`, `permfails`, `transfails`, and `deliveries` fields respectively.
+
+Use this request with the `domain=a.specific.domain` argument to show the statistics for a specific domain. The `binding` and the `ip` argument can be used in the same way, but they're exclusive and the `ip` argument will be ignored in case both are specified. The output cannot be limited or split into multiple pages.
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/queues-mailq.md:63]: Let's remove mention if the ip parameter.

File: content/momentum/4/http-api-stats/queues-summary-sizes.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/queues-summary-sizes.md:1]: We don't need to document this endpoint.  The queue sizes can be retrieved from /stats/summary which supersedes this endpoint. 
(we still need to go back in sparkpost circonus agent and get queue sizes from /stats/summary).

File: content/momentum/4/http-api-stats/summary.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+
+The average number of DNS queries/second performed since startup or last summary reset.
+
+</dd>
+
+<dt>ResidentMessages</dt>
+
+<dd>
+
+TODO
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/summary.md:149]: The number of message bodies held in memory.

File: content/momentum/4/http-api-stats/summary.md [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+
+</dd>
+
+<dt>DiskQueueProgress</dt>
+
+<dd>
+
+TODO
+
+The progress of the message queue being written to the disk: the current amount and the total/target amount.
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/summary.md:159]: I think it's sufficient to just say "The message spool in progress"

File: content/momentum/4/http-api-stats/summary.md [hunk 8]
```diff
@@ -0,0 +1,261 @@
...
+
+</dd>
+
+<dt>TransientFailures</dt>
+
+<dd>
+
+TODO
+
+The number of outbound messages transiently failed since it was started or since the last summary reset.
```
Comment (cjarnold) [inline content/momentum/4/http-api-stats/summary.md:185]: Seems good to me

**Review by rafaelmatsumotomb — APPROVED** [review]

**Review by rafaelmatsumotomb — APPROVED** [review]
